CMakeLists.txt

```diff
@@ -69,14 +69,31 @@ else()
   endif()
 endif()
 
-# todo: support find_package(bgfx)
-if(NOT bgfx_FOUND)
+option(OPENBLACK_BUILD_BGFX
+"Build bgfx alongside project (only use for developing bgfx)" OFF)
+
+if(NOT OPENBLACK_BUILD_BGFX)
+  find_package(bgfx CONFIG)
+  if(NOT bgfx_DIR)
+  if (WIN32)
+    message(STATUS "Precompiled bgfx binaries can be found here: https://github.com/openblack/bgfx.cmake/releases/tag/latest")
+  endif()
+
+    message(FATAL_ERROR " \
+Failed to find bgfx, either install from: https://github.com/openblack/bgfx.cmake \
+Or set the variable OPENBLACK_BUILD_BGFX to build it with the project. \
+")
+  endif()
+else()
+  message(AUTHOR_WARNING "Using OPENBLACK_BUILD_BGFX, if you are not developing use an install target instead.")
```

Comment:
Getting this message. Seems a little redundant since the dev is the one setting this variable
```
CMake Warning (dev) at CMakeLists.txt:88 (message):
  Using OPENBLACK_BUILD_BGFX, if you are not developing use an install target
  instead.
```

Reply:
yeah just wanna make sure people aren't using it without a good reason